docs/docs/guides/installation/get-started/linux.mdx

```diff
@@ -0,0 +1,17 @@
+## Download The CockroachDB binary
+
+Download a `cockroach` binary as described [in the CockroachDB docs](https://www.cockroachlabs.com/docs/v22.1/install-cockroachdb).
+ZITADEL is tested against CockroachDB v22.1.0 and Ubuntu 20.04.
+
+## Run CockroachDB and ZITADEL
+
+```bash
+# Run a CockroachDB instance
+cockroach start-single-node --insecure --background --http-addr :9090
+
+# Download the zitadel binary
+curl -s https://api.github.com/repos/zitadel/zitadel/releases/tags/v2.0.0-v2-alpha.24 | grep "browser_download_url.*zitadel_Linux_$(uname -i).tar.gz" | cut -d '"' -f 4 | xargs wget -qO - | sudo tar --extract --gzip --overwrite --directory /usr/local/bin zitadel && sudo chown $(id -u):$(id -g) /usr/local/bin/zitadel
+
+# Run the zitadel binary
+ZITADEL_EXTERNALSECURE=false zitadel start-from-init --masterkey "MasterkeyNeedsToHave32Characters"
```

Comment:
Do we need to add --tlsMode disabled here as well?

Reply:
yes, done